docs/content/topics/developers.md

```diff
@@ -8,47 +8,49 @@ aliases:
   - /topics/developers.md
 ---
 
-TODO: update per v2
-
 # Developer Guide
 
 This document explains how to get started developing Brigade.
 
-Brigade is composed of numerous parts.  The following represent the core components:
+Brigade is composed of numerous parts.  The following represent the core
+components:
 
-- brigade-controller: The Kubernetes controller for delegating Brigade events
-- brigade-worker: The JavaScript runtime for executing `brigade.js` files. The
-  controller spawns these, though you can run one directly as well.
-- brigade-api: The REST API server for user interfaces
-- brigade-project: The Helm [chart][brigade-project-chart] for installing Brigade projects
-- brigade-vacuum: The stale build cleaner-upper (optional; enabled by default)
+- apiserver: Brigade's API server, where the majority of core Brigade
+  logic lives
+- scheduler: Schedules event workers and jobs on the underlying substrate
+- observer: Observes (and records) state changes in workers and jobs and
+  ultimately cleans them up from the underlying substrate
+- worker: The default runtime for executing `brigade.js/.ts` files.
+- logger: Event log aggregator, with Linux and Windows variants
 - brig: The Brigade CLI
-- git-sidecar: The code that runs as a sidecar in cluster to fetch Git repositories (optional; enabled by default)
+- git-initializer: The code that runs as a sidecar to fetch Git repositories
+  for vcs-enabled projects
 
-Additionally, there are several opt-in gateways that can be enabled via Helm chart values.  These are:
+Additionally, there are several opt-in gateways that can be deployed alongside
+Brigade.  These are:
```

Comment:
We no longer include any gateways in the base repo / base chart.

Reply:
Ok, removed this section.